package.json

```diff
@@ -127,7 +127,7 @@
     "build-prod-min": "rollup -c --environment BUILD:production,MINIFY:true",
     "build-csp": "rollup -c rollup.config.csp.js",
     "build-query-suite": "rollup -c test/integration/rollup.config.test.js",
-    "build-flow-types": "cp build/mapbox-gl.js.flow dist/mapbox-gl.js.flow && cp build/mapbox-gl.js.flow dist/mapbox-gl-dev.js.flow",
+    "build-flow-types": "mkdir -p dist && cp build/mapbox-gl.js.flow dist/mapbox-gl.js.flow && cp build/mapbox-gl.js.flow dist/mapbox-gl-dev.js.flow",
```

Comment:
The `build-flow-types` build step is also used in our CI build pipeline, but in a different order:

https://github.com/mapbox/mapbox-gl-js/blob/ced93c1624e37e618024f621a1744ff24b279c26/.circleci/config.yml#L152-L167

I suggest moving this into the `publish-clean-check` step. It belongs there as part of the `clean` up

Reply:
I prefer to leave it here so it's not order dependent like in our CI; the second part of this command depends on `dist` already existing since it copies into it. Our CI runs `yarn run build-style-spec` [which creates `dist/style-spec/`](https://github.com/mapbox/mapbox-gl-js/pull/9590/files/f8708f37d12bc5e1e0e2016a16843f7a2ad8a8d0#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R132) folder and `yarn run build-flow-types` runs after, so this folder happens to exist in this order only with a clean checkout.